### PR TITLE
feat: toast component

### PR DESCRIPTION
**Pull request by tusharkandpal**
To Check:
✅ type includes: basic & stacked toast
✅ added styles for the both

Recording 2022-03-27 at 11 37 53

### CODE REVIEW

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+/* || STACKED TOAST */
+.toast-stacked {
+  flex-direction: column;
+}
+
+.toast-text,
+.toast-actions {
+  margin: 0.5rem 0;
```
Comment (psychoactiv) [inline components/toast/toast.css:17]: suggestion - you can use px for values less than 1 rem

**Review by psychoactiv — APPROVED** [review]
clean code, loooks good

File: components/toast/toast.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../text-utilities/text-utilities.html" class="sidebar-link"
+              >Text Utilities</a
+            >
+          </li>
+          <li
+            class="sidebar-item mp-m active-component"
+            style="background-color: var(--toast-color)"
+          >
```
Comment (fahd23) [inline components/toast/toast.html:78]: Why inline styling, you can avoid it right?

**Review by fahd23 — APPROVED** [review]
Looks good

File: components/toast/toast.html (hunk 2 above)
Comment (tusharkandpal) [reply components/toast/toast.html:78]: No.. unfortunately 😅

File: components/toast/toast.css (hunk 1 above)
Comment (tusharkandpal) [reply components/toast/toast.css:17]: Just being consistent throughout the project. Hope its fine.

**Review by neetukumari4858 — APPROVED** [review]
code is readable good to see

**Pull request merged**